cc33conf/suite/trixie/debian/rules

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --sourcedir=cc33conf
```

Comment:
Do you need `--buildsystem=cmake --builddir=cc33conf/build` here?

Reply:
Yes you're right, I have updated the PR.